components/integration/base/delete-snapshots.yaml

```diff
@@ -0,0 +1,42 @@
+#kind: Role
```

Comment:
Why those line were commented out?

Reply:
Commented them out for testing, forgot to uncomment.  Fixed now